src/Metadata/ConventionModelMapper.cs

```diff
@@ -23,6 +23,7 @@ public partial class ConventionModelMapper : IModelMapper, IModelMapperInternal
         private const string ConventionalPrimaryKeyName = "ID";
 
         private readonly Dictionary<Type, EntityMapper> _entities = new Dictionary<Type, EntityMapper>();
+        private readonly Dictionary<string, Expression> _queryFilters = new Dictionary<string, Expression>();
```

Comment:
It would probably be better if the key was case insensitive (`OrdinalIgnoreCase`).